client/lib/post-normalizer/rule-content-make-embeds-safe.js

```diff
@@ -49,7 +49,7 @@ export default function makeEmbedsSafe( post, dom ) {
 		} else if ( iframeIsAllowed( iframe ) ) {
 			iframe.setAttribute( 'sandbox', 'allow-same-origin allow-scripts allow-popups' );
 		} else {
-			iframe.setAttribute( 'sandbox', '' );
+			iframe.setAttribute( 'sandbox', 'allow-scripts' );
```

Comment:
Enabling `sandbox="allow-scripts"` to any iframe without verifying it's from a safe source is definitely a security risk. I would advise strongly against it. 

I'm in favor of the alternative idea you suggested - introducing a fading overlay at the bottom of the frame.

Reply:
Since the iFrame is sandboxed and doesn't allow access to parent frame, I am not sure if `sandbox="allow-scripts"` is a security risk. It can certainly be:

- a performance hog
- behave incorrectly (itself)

The parent frame content and cookies _should_ be safe. 

We could also limit the `iFrames` having `sandbox="allow-scripts"` only to those ending with `/embed` which is the format for WordPress embeds eg https://azcir.org/news/2021/06/14/504-plans-favor-wealthy-white-students-disabilities/embed/.

Just doing a final stretch, before going on with the CSS hack 😃

Reply:
I agree, the parent frame content and cookies are safe if we don't provide `allow-same-origin` too, but it just feels wrong (and too permissive) to allow any site to execute any script they wish. There's a bunch one can do with JavaScript (even without access to the parent frame) and I'm not sure allowing this is justified when we can just go around with a visual trick. Anyway, I don't have super strong feelings about this, and I'm generally open about it.